tests/integration/targets/aci_access_spine_interface_selector_to_spine_switch_profile/tasks/main.yml

```diff
@@ -0,0 +1,162 @@
+# Test code for the ACI modules
+# Copyright: (c) 2017, Bruno Calogero <brunocalogero@hotmail.com>
+# Adapted from aci_interface_selector_to_switch_policy_leaf_profile
+# Copyright: (c) 2023, Eric Girard <@netgirard>
+# GNU General Public License v3.0+ (see LICENSE or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+- name: Test that we have an ACI APIC host, ACI username and ACI password
+  fail:
```

Comment:
Can we use full module names?
ie. ansible.builtin.fail,
ansible.builtin.set_fact,
ansible.builtin.assert
ect.